crates/oxc_linter/src/rules/unicorn/prefer_spread.rs

```diff
@@ -231,7 +232,9 @@ fn is_not_array(expr: &Expression, ctx: &LintContext) -> bool {
         _ => return false,
     };
 
-    if ident.starts_with(|c: char| c.is_ascii_uppercase()) && ident.to_ascii_uppercase() != ident {
+    if ident.starts_with(|c: char| c.is_ascii_uppercase())
+        && ident.cow_to_ascii_uppercase() != ident
+    {
```

Comment:
probably don't need to use `cow` here as we are just comparing the strings?

or maybe we do 🤷

Reply:
I'm not sure😵‍💫, will there be a slight performance improvement? Or has there been no improvement at all?